public/index.js

```diff
@@ -17,6 +18,62 @@ let textContent;
 let imageUrl;
 let isLactoseIntolerant;
 let dishOriginCountry;
+const defaultRecipe = `
```

Comment:
Default recipe. This isn't as refined yet but it provides a default message and recipe.